UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentView.cs

```diff
@@ -7,8 +7,34 @@ namespace PlayFab
 
     public class PlayFabMultiplayerAgentView : MonoBehaviour
     {
+        /// <summary>
+        /// Static reference to the current active instance of the PlayFabMultiplayerAgentView.
+        /// </summary>
+        public static PlayFabMultiplayerAgentView current { get; private set; };
+        
         private float _timer;
 
+        /// <summary>
+        /// Awake constructor
+        /// </summary>
+        private void Awake()
+        {
+            Debug.Log($"{Time.fixedTime} PlayFabMultiplayerAgentView awake ");
```

Comment:
can we have `nameof(PlayFabMultiplayerAgentView)` for the (unlikely) event we ever change the class name?